rom/laghos.cpp

```diff
@@ -1253,15 +1252,14 @@ int main(int argc, char *argv[])
                             else
                                 outfile_twp << cutoff[0] << ", " << cutoff[1] << ", " << cutoff[2] << "\n";
                         }
-                        delete sampler;
+                        sampler = NULL;
```

Comment:
Replacing `delete sampler` with `sampler = NULL` looks like a memory leak (`sampler` will never be deleted). Should there be `delete sampler` and `sampler = NULL`?

Reply:
Yes, `delete sampler` should be kept.